index.html

```diff
@@ -4221,11 +4221,18 @@ <h3>HTML Attribute State and Property Mappings</h3>
               <td class="elements">
                 <a data-cite="html/interaction.html#attr-contenteditable">HTML elements</a>
               </td>
-              <td class="aria">?</td>
+              <td class="aria">
+                <p class="general">
+                  If set to an element which already exposes a corresponding role, then no specific ARIA mapping.
```

Comment:
```suggestion
                  If set on an element which already exposes a role, then no specific ARIA mapping.
```

Notes:
- I find the "set to" wording a little odd because I am expecting a _value_ to follow the word "to" (so I changed it to "set on")
- Alternatively, did you want to say, 'If set to "true" or "" on an element...'
- Not quite sure whether "corresponding" is needed?

Reply:
- "set on" is fine by me.
- yeah, that should be more clear... while "true" and "" are the same...but since "false" matters here, it should really say "if contenteditable is true"
- i did mean to use "corresponding" per its use in other parts of the spec.  we can't just say "which already exposes a role", because "generic" is a role, and that's what we're trying to mitigate against here.  So this really should be something like "if set on an element which exposes an implicit aria role, other than generic, then no specific ARIA mapping." - to indicate that `contenteditable` has no other ARIA mapping itself